lib/theme_mode_handler.dart

```diff
@@ -43,29 +50,21 @@ class ThemeModeHandlerState extends State<ThemeModeHandler> {
   @override
   void initState() {
     super.initState();
-    _loadThemeMode().then((ThemeMode value) {
+    _loadThemeMode().then((value) {
```

Comment:
Extract the setState to outside of initState to keep single responsability.

then(_handleLoadThemeMode);
_handleLoadThemeMode(value) => setState...;